java-sdk-example/src/test/java/CreateSandboxTestWithCustomPatch.java

```diff
@@ -0,0 +1,136 @@
+import com.signadot.ApiClient;
+import com.signadot.ApiException;
+import com.signadot.api.SandboxesApi;
+import com.signadot.model.*;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+public class CreateSandboxTestWithCustomPatch {
+
+  public static final String HOTROD = "hotrod";
+  public static final String CUSTOM_PATCH = "spec:\n" +
+                                            "  template:\n" +
+                                            "    spec:\n" +
+                                            "      containers:\n" +
+                                            "      - name: hotrod\n" +
+                                            "        env:\n" +
+                                            "        - name:  PATCH_TEST_VAR\n" +
+                                            "          value: foo\n";
+
+  public static final String ORG_NAME = System.getenv("SIGNADOT_ORG");
+  public static final String SIGNADOT_API_KEY = System.getenv("SIGNADOT_API_KEY");
+  public static final String CLUSTER_NAME = System.getenv("SIGNADOT_CLUSTER_NAME");
+
+  private static RequestSpecification requestSpec;
+
+  ApiClient apiClient;
+  SandboxesApi sandboxesApi;
+  CreateSandboxResponse response;
+  String sandboxID;
+
+  @BeforeSuite
+  public void createSandboxWithCustomPatch() throws ApiException, InterruptedException {
```

Comment:
Some explanation/documentation on what we're doing in this test, including the test setup (how we're setting the env var from custom patch and validating the working of it.)